kustomize/commands/edit/add/addresource.go

```diff
@@ -35,6 +36,9 @@ func newCmdAddResource(fSys filesys.FileSystem) *cobra.Command {
 			return o.RunAddResource(fSys)
 		},
 	}
+	cmd.Flags().BoolVar(&o.skipValidation, "skip-validation", false,
```

Comment:
For parity with the flag we are adding to kustomize localize (https://github.com/kubernetes-sigs/kustomize/issues/5276, can we likewise name this flag `--no-verify`?

Reply:
That would be better. I've renamed the flag.